catalog/ui/catalog-ui-search/src/main/webapp/js/model/User.js

```diff
@@ -273,6 +273,7 @@ define([
                 preferences: new User.Preferences(),
                 isGuest: true,
                 username: 'guest',
+                userid: 'guest',
```

Comment:
I wonder if we could update it to just set the id as whatever the userid is.  I'm not sure why all the ids are set to 'user'.